Source/DafnyLanguageServer/Language/DafnyLangParser.cs

```diff
@@ -72,9 +73,22 @@ public sealed class DafnyLangParser : IDafnyParser, IDisposable {
         if (parseErrors != 0) {
           logger.LogDebug("encountered {ErrorCount} errors while parsing {DocumentUri}", parseErrors, document.Uri);
         }
+
         if (!TryParseIncludesOfModule(program.DefaultModule, program.BuiltIns, errorReporter, cancellationToken)) {
           logger.LogDebug("encountered error while parsing the includes of {DocumentUri}", document.Uri);
         }
+
+        return program;
+      } catch (Exception e) {
+        logger.LogDebug(e, "encountered an exception while parsing {DocumentUri}", document.Uri);
+        var internalErrorDummyToken = new Token {
+          filename = document.GetFilePath(),
+          line = 1,
+          col = 1,
+          pos = 0,
+          val = " "
```

Comment:
What's the reason for this space? Wouldn't an empty string, i.e. `string.Empty`, work?

Reply:
I previously thought that an empty val would prevent the message from being displayed, but that happened only if the line was 0. With the line 1 and the string empty, it works (and it still underlines the first character of the file in red), so I'm pushing this that @atomb also asked for.